pages/common/git-authors.md

```diff
@@ -0,0 +1,17 @@
+# git authors
+
+> Display log or list of committers of a Git repository. Everything except `--list` will create a new file called `AUTHORS`.
+> Part of `git-extras`.
+> More information: <https://github.com/tj/git-extras/blob/master/Commands.md#git-authors>.
+
+- Create or replace the `AUTHORS` file with a list of committers and open it in the default editor:
+
+`git authors`
+
+- Print a full list of committers:
+
+`git authors --list`
+
+- List committers, excluding emails, in the default editor, reading from the `AUTHORS` file:
```

Comment:
Perhaps this will read better?
```suggestion
- Launch the default editor to display a list of committers (excluding emails) from the `AUTHORS` file:
```

Reply:
I disagree with this one, it sounds wrong

Reply:
Maybe sounds wrongs, but @sbrl is right, is not a list in terminal, Launch an app with something. :thinking: @bl-ue ?

Reply:
To me they both imply basically the same meaning.

Reply:
I think too. When you list something in the default editor, I think that implies that it launches it first. I prefer the first one.

Reply:
Hrm, I see the concerns here. However this PR has been prematurely merged, which limits the usefulness of me suggesting anything else here. Regardless, what about this?
```suggestion
- Display a list of committers (excluding emails) from the `AUTHORS` file in the default text editor:
```